### DISCUSSION TITLE
How is database migration handled in Redash right now?

### DISCUSSION THREAD

**Discussion by zachliu** [getredash/redash#7005] [Q&A]
I've been using the Mozilla fork since 2019. They added `/app/manage.py db upgrade` to the server/worker/scheduler entrypoint. When I tried the same thing on `24.06.0-dev`, the database was successfully upgraded ~but then the server container exited with code 0~

From the [documentation](https://redash.io/help/open-source/admin-guide/how-to-upgrade) and this discussion https://github.com/getredash/redash/discussions/6567, looks like the protocol at the moment is executing `./bin/docker-entrypoint manage db upgrade` manually on the server. Am I correct?

Since I'm using k8s and [k9scli](https://k9scli.io/), I just shell into a running container and execute `./bin/docker-entrypoint manage db upgrade`, which is not a big deal. But I am looking forward to an automated db migration mechanism.

**Comment by BowlesCR** [comment]
Do you know about the helm chart? https://github.com/getredash/contrib-helm-chart/
It has an upgrade hook that runs the migration as a k8s job.
